### DISCUSSION TITLE
not found plots chia network

### DISCUSSION THREAD

**Discussion by jlopez-hub** [Chia-Network/chia-blockchain#8317] [Support]
Hello there

My name is Jose and I lost the MNEMONICS (24 words) because it was recorded on a tablet that had a little big accident.

I created a new wallet since to date I did not have any reward in my wallet, but now I have the problem that I cannot find any plot created to date, I have more than 200Tb in plots.

I have reinstalled chia blockchain and even the linux mint operating system in case there was any file that was giving problems, but chia has not worked, I have also deleted all the chia keys and I have created it again and it has not worked either.

the version of chia that I am using is 1.2.5, and the plots were created with 1.1.5. I have also returned to the previous version chia 1.1.5 without results

I would be very grateful if someone could give me any suggestions to solve the problem.

**Comment by loppefaaret** [comment]
uh, if you do not have the key imported anywhere, and you have lost the 24 words - all the XCH in the wallet, and any plots tied to the key are lost...

**Comment by BrandtH22** [comment]
@jlopez-hub , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
